### DISCUSSION TITLE
Keyboard shortcurts

### DISCUSSION THREAD

**Discussion by rbn30117** [git-for-windows/git#3872] [Q&A] [answered]
Why are keyboard shortcuts like CTRL+V not supported?

**Comment by dscho** [comment, marked as answer]
Are you talking about Git Bash? If so, Ctrl+V has an already-established, very different meaning there. For a full explanation, see e.g. [this StackOverflow thread](https://superuser.com/questions/421463/why-does-ctrl-v-not-paste-in-bash-linux-shell).
